### PR TITLE
Bpk 2664 document missing android components

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

## TODO
 - [x] Interactive star rating

### CODE REVIEW

File: docs/src/layouts/links.js [hunk 1]
```diff
@@ -217,7 +241,7 @@ const ComponentsLinks = [
       {
         id: 'STAR_RATING',
         route: routes.STAR_RATING,
-        children: 'Star rating',
+        children: 'Star Rating',
```
Comment (georgegillams) [inline docs/src/layouts/links.js]:
```suggestion
        children: 'Star rating',
```

File: docs/src/layouts/links.js [hunk 2]
```diff
@@ -57,6 +57,30 @@ const ComponentsLinks = [
...
+      {
+        id: 'CHECKBOX',
+        route: routes.CHECKBOX,
+        children: 'Checkbox',
+        tags: ['android'],
+      },
+      {
+        id: 'STAR_RATING_INTERACTIVE',
+        route: routes.STAR_RATING_INTERACTIVE,
+        children: 'Star Rating: Interactive',
```
Comment (georgegillams) [inline docs/src/layouts/links.js]:
```suggestion
        children: 'Star rating: interactive',
```

File: docs/src/pages/AndroidHorizontalNavPage/AndroidHorizontalNavPage.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+import React from 'react';
+
+import readme from '../../../../backpack-android/docs/HorizontalNav/README.md';
+import screenshotDefault from '../../../../backpack-android/docs/HorizontalNav/screenshots/default.png';
+import DocsPageBuilder from '../../components/DocsPageBuilder';
+
+const components = [
+  {
+    id: 'default',
```
Comment (georgegillams) [inline docs/src/pages/AndroidHorizontalNavPage/AndroidHorizontalNavPage.js]: I would call this something like "All varieties" as we're showing more than the "default" configuration in this screenshot

File: docs/src/pages/AndroidStarRatingInteractivePage/AndroidStarRatingInteractivePage.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import readme from '../../../../backpack-android/docs/InteractiveStarRating/README.md';
+import screenshotDefault from '../../../../backpack-android/docs/InteractiveStarRating/screenshots/default.png';
+import DocsPageBuilder from '../../components/DocsPageBuilder';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb:
+      'An interactive version of Star Rating. These can be set at half star intervals. ', // eslint-disable-line max-len
```
Comment (georgegillams) [inline docs/src/pages/AndroidStarRatingInteractivePage/AndroidStarRatingInteractivePage.js]: Can we make the `Star rating` text link to the Star rating page?

Here's an example of how to do it!
https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/pages/AlignmentPage/AlignmentPage.js#L69-L74

File: docs/src/pages/AndroidStarRatingInteractivePage/AndroidStarRatingInteractivePage.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        altText: 'Android Interactive Star Rating Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+];
+
+const AndroidStarRatingPage = ({ ...rest }) => (
+  <DocsPageBuilder
+    title="Star Rating Interactive"
```
Comment (georgegillams) [inline docs/src/pages/AndroidStarRatingInteractivePage/AndroidStarRatingInteractivePage.js]:
```suggestion
    title="Star rating interactive"
```

File: docs/src/pages/AndroidStarRatingPage/AndroidStarRatingPage.js [hunk 6]
```diff
@@ -41,13 +41,13 @@ const components = [
   },
 ];
 
-const NativeStarRatingPage = ({ ...rest }) => (
+const AndroidStarRatingPage = ({ ...rest }) => (
   <DocsPageBuilder
-    title="Star Ratings"
+    title="Star Rating"
```
Comment (georgegillams) [inline docs/src/pages/AndroidStarRatingPage/AndroidStarRatingPage.js]:
```suggestion
    title="Star rating"
```

File: docs/src/pages/CheckboxPage/CheckboxPage.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+import DocsPageWrapper from '../../components/DocsPageWrapper';
+import IntroBlurb from '../../components/IntroBlurb';
+import AndroidToastPage from '../AndroidCheckboxPage';
+
+const Page = () => (
+  <DocsPageWrapper
+    title="Checkbox"
+    blurb={[
+      <IntroBlurb>Checkbox is a Material Design UI component.</IntroBlurb>,
```
Comment (georgegillams) [inline docs/src/pages/CheckboxPage/CheckboxPage.js]:
```suggestion
      <IntroBlurb>Checkboxes allow the user to select one or more items from a set.</IntroBlurb>,
```

File: docs/src/pages/StarRatingInteractrivePage/StarRatingInteractrivePage.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+import React from 'react';
+
+import DocsPageWrapper from '../../components/DocsPageWrapper';
+import IntroBlurb from '../../components/IntroBlurb';
+import AndroidToastPage from '../AndroidStarRatingInteractivePage';
+
+const Page = () => (
+  <DocsPageWrapper
+    title="Star Rating Interactive"
```
Comment (georgegillams) [inline docs/src/pages/StarRatingInteractrivePage/StarRatingInteractrivePage.js]:
```suggestion
    title="Star rating interactive"
```

File: docs/src/pages/ToastPage/ToastPage.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+import React from 'react';
+
+import DocsPageWrapper from '../../components/DocsPageWrapper';
+import IntroBlurb from '../../components/IntroBlurb';
+import AndroidToastPage from '../AndroidToastPage';
+
+const Page = () => (
+  <DocsPageWrapper
+    title="Toast"
+    blurb={[<IntroBlurb>Toast is a Material Design UI component.</IntroBlurb>]}
```
Comment (georgegillams) [inline docs/src/pages/ToastPage/ToastPage.js]:
```suggestion
    blurb={[<IntroBlurb>Toasts provide brief messages about app processes at the bottom of the screen.</IntroBlurb>]}
```

File: package.json [hunk 10]
```diff
@@ -41,7 +41,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "prepare-commit-msg": "jirabot precommit $HUSKY_GIT_PARAMS || true"
```
Comment (georgegillams) [inline package.json]: 😜

File: docs/src/pages/AndroidHorizontalNavPage/AndroidHorizontalNavPage.js (hunk 3 above)
Comment (bvitaliyg) [reply docs/src/pages/AndroidHorizontalNavPage/AndroidHorizontalNavPage.js]: Done

**Pull request merged**